### PR TITLE
Dev 02

### PR DESCRIPTION
**Pull request by mjfrigaard**
I kept all the changes in dev-02

### CODE REVIEW

File: purrr_basics/purrr_basics.Rmd [hunk 1]
```diff
@@ -809,23 +1204,51 @@ class: middle, center
 
 ---
 
-## Your turn 9
+## `group_walk()`
+
+### Use `group_walk()` for side effects across groups 
+
+```{r group_walk}
+fs::dir_create(temp <- "temp")
```
Comment (malcolmbarrett) [inline purrr_basics/purrr_basics.Rmd:1212]: let's move `temp <- "temp"` to the line before; I prefer not to tuck away assignment actions

File: purrr_basics/purrr_basics.Rmd [hunk 2]
```diff
@@ -809,23 +1204,51 @@ class: middle, center
...
+gapminder %>%
+  group_by(continent) %>%
+  group_walk(~ write_csv(x = .x, file = file.path(temp,
```
Comment (malcolmbarrett) [inline purrr_basics/purrr_basics.Rmd:1215]: let's vary this a bit and do `group_walk(write_csv, [...the rest])`

File: purrr_basics/purrr_basics.Rmd [hunk 3]
```diff
@@ -126,59 +129,366 @@ gapminder[1:6, "pop"]
...
+```
+
+---
+
+## `mutate(across())` & `summarise()`
+
+```{r slide-24-code, eval=FALSE}
+gapminder %>% 
+  group_by(continent) %>% 
+    summarise(across(c("lifeExp", "gdpPercap"), list(med = median, #<<
```
Comment (malcolmbarrett) [inline purrr_basics/purrr_basics.Rmd:330]: let's vary this one a bit, too, and do `summarise(across(c("lifeExp", "gdpPercap"), median)` to show we don't always need a list

File: purrr_basics/purrr_basics.Rmd [hunk 4]
```diff
@@ -126,59 +129,366 @@ gapminder[1:6, "pop"]
...
+---
+
+## `mutate(across())` & `summarise()`
+
+### `tidyselect()` helpers and named columns
+
+```{r slide-27-mutate-across-summarise-code, eval=FALSE}
+diabetes %>% 
+  group_by(gender) %>% 
+    summarise(across(starts_with("bp"), list(mean = mean))) #<<
```
Comment (malcolmbarrett) [inline purrr_basics/purrr_basics.Rmd:397]: Let's just do `mean` for these instead of `list(mean = mean)`.

**Review by malcolmbarrett — CHANGES_REQUESTED** [review]
Looking good! Just some minor tweaks

**Comment by malcolmbarrett** [comment]
FYI I tweaked these myself. I'll merge shortly. Thanks!

File: purrr_basics/purrr_basics.Rmd (hunk 4 above)
Comment (mjfrigaard) [reply purrr_basics/purrr_basics.Rmd]: I can do this, but the output isn't very clear: 

```
## # A tibble: 2 × 5
##   gender `1_bp.1s` `1_bp.1d` `1_bp.2s` `1_bp.2d`
##   <chr>      <dbl>     <dbl>     <dbl>     <dbl>
## 1 female      136.      82.5      153.      91.8
## 2 male        138.      84.5      151.      93.5
```

Should I swap `{.fn}` and `{.col}`?

File: purrr_basics/purrr_basics.Rmd (hunk 2 above)
Comment (mjfrigaard) [reply purrr_basics/purrr_basics.Rmd]: This is the only one I am fumbling with...I still can't convert it to the standard format. I am struggling on referencing `.data` in the `...` arguments. Would love to see how you got this to work.

**Pull request merged**